File: Easy/problem_3/sol.py
def factorial(num):
    if num == 0 or num < 1:
        raise Exception(
            "Invalid Input, 0 or Negative numbers are not allowed.")
    else:
        factorial = 1
        for i in range(1, num+1):
            factorial *= i
        return factorial

# better and faster approach to use math.factorial(number)


output = factorial(1)
print(output)

File: Easy/problem_3/sol.js
function factorial(num) {
  if (num === 1) return 1;
  else if (num === 0) {
    throw new Error("Error! Factorial of 0 is not valid.");
  } else if (num < 0) {
    throw new Error("Error! Factorial of Negative numbers is not valid.");
  } else {
    let factorial = 1;
    for (let i = 1; i <= num; i++) {
      factorial *= i;
    }
    return factorial;
  }
}
